### PR TITLE
HOMS-289 Move dev installation instruction into separate files

### PR DESCRIPTION
**Pull request by vladdoncheg**

### CODE REVIEW

**Comment by vladdoncheg** [comment]
I think you will feel yourself more comfortable reading docs directly:
https://github.com/latera/homs/blob/HOMS-289/README.md
https://github.com/latera/homs/blob/HOMS-289/WITHOUT_ORACLE.md
https://github.com/latera/homs/blob/HOMS-289/WITH_ORACLE.md

File: CHANGELOG.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ v1.8.0 [unreleased]
 - [#351](https://github.com/latera/homs/pull/351) Improve claim button styles on widget form.
 - [#354](https://github.com/latera/homs/pull/354) Wrap task short pop up list page together.
 - [#353](https://github.com/latera/homs/pull/353) Integrate new claiming components.
+- [#355](https://github.com/latera/homs/pull/355) Move dev installation instruction into separate files.
```
Comment (TimAle) [inline CHANGELOG.md]: I don't think we can call that feature :)
Don't mention that in changelog.

File: WITHOUT_ORACLE.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+### With Oracle Instant Client
```
Comment (mxsmirnoff) [inline WITHOUT_ORACLE.md]: First line in file should be a top level heading

File: WITHOUT_ORACLE.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+If you want to use Oracle as source for your HOMS instance.
+
+1. Install [docker-compose](https://docs.docker.com/compose/install/).
+
+2. Clone HOMS git repository:
+
+  ```
```
Comment (mxsmirnoff) [inline WITHOUT_ORACLE.md]: Fenced code blocks should have a language specified

File: README.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
-# Hydra Order Management System (homs)
+# Hydra Order Management System (HOMS)
 
-*homs* is an open source web application for order and business process management.
+*HOMS* is an open source web application for order and business process management.
 
 Application consists of three parts:
 * User interface - managing orders and tasks for registered users (Single Page Application).
```
Comment (mxsmirnoff) [inline README.md]: Lists should be surrounded by blank lines

File: README.md [hunk 5]
```diff
@@ -16,148 +16,58 @@ Resources:
...
-11. (Optional) If you want to use demo processes navigate to [Camunda](http://localhost:8080/camunda/app/admin/default/#/user-create) and create user with `user@example.com` email.
-
-12. Login at [HydraOMS](http://localhost:3000) with *`user@example.com`*/*`changeme`*.
-
-#### In development
-
-If you don't want to use Oracle as source for your HOMS instance:
 
-##### Without Oracle Instant Client
+> :pushpin: All variables are set in `.env` file. There you can change them, if you want to.
```
Comment (mxsmirnoff) [inline README.md]: Why quote block?

File: WITHOUT_ORACLE.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+8. Place the downloaded Oracle Instant Client RPMs in the same directory as the `Dockerfile` and run:
+
+    ```bash
+    docker build -t latera/homs-with-oracle -f Dockerfile.oracle .
+    ```
+
+9. Update `config/sources.yml` file with database credentials
+
+    ```bash
```
Comment (mxsmirnoff) [inline WITHOUT_ORACLE.md]: yml

File: WITHOUT_ORACLE.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+    TNSNAMES_PATH=/dir/with/tnsnames.ora
+    ```
+
+    :pushpin: For access to host machine OS X users can use special DNS name `host.docker.internal` as host in `tnsnames.ora` ([details](https://docs.docker.com/docker-for-mac/networking)).
+
+11. Add test environment to `config/database.yml`:
+
+    ```bash
```
Comment (mxsmirnoff) [inline WITHOUT_ORACLE.md]: yml

**Pull request merged**